tools/disasm/variables.txt

```diff
@@ -10997,9 +10997,6 @@
     0x80A0C55C:("D_80A0C55C","f32","",0x4),
     0x80A0C560:("D_80A0C560","f32","",0x4),
     0x80A0C7C0:("En_Boj_05_InitVars","UNK_TYPE1","",0x1),
-    0x80A0C7E0:("enBoj05OverlayInfo","OverlayRelocationSection","",0x14),
-    0x80A0C7F4:("enBoj05OverlayRelocations","u32","[4]",0x10),
-    0x80A0C80C:("enBoj05OverlayInfoOffset","u32","",0x4),
```

Comment:
These modifications were intentional right? This feels like a different PR

Reply:
Yeah this is intentional, these are wrong symbols that shouldn't be in there. Anghelo figured it out and said I should remove them.